components/src/core/components/Input/Select/select-input.scss

```diff
@@ -220,23 +219,51 @@
   border-radius: 0.85rem;
   &.button-double-line {
     padding: 0.5rem 0.75rem 0.5rem 1rem;
+    :deep(.oxd-button-label-wrapper) {
+      display: block;
+      overflow: initial;
+      white-space: initial;
+      text-overflow: initial;
+      .label-double-line-wrapper.label-double-line .label {
+        overflow: initial;
+        text-overflow: initial;
+        white-space: initial;
+        word-break: break-word;
+        text-align: left;
+        padding-right: 5px;
+      }
+    }
   }
 }
 .oxd-icon {
   &.oxd-button-icon-right {
     margin-left: 0.5rem;
   }
 }
+.mini-icon-left {
+  margin-right: 0.5rem;
```

Comment:
We should not change Select input under this change.

Reply:
We should not change Select input under this change.